add org README (#1)
oracle/.github: profile/README.md

> ![](ogho_banner.png)

Reviewer: ```suggestion
![Oracle Logo](ogho_banner.png "Open source at Oracle. Everyone is invited. Let's collaborate together")
```

# Open Source at Oracle

Oracle is the purveyor of the industry’s most widely adopted open source technologies, such as Java and MySQL, and one of the founding members of the Linux Foundation, the Eclipse Foundation, and the Java Community Process.

But what has guided us along the way?

> Our mission is to empower developers and continue supporting open source technologies. Through Oracle Cloud Infrastructure and its 37 cloud regions, we’re providing more opportunities than ever to make open source technology available to everyone.<br><br>

Reviewer: ```suggestion
Our mission is to empower developers and continue supporting open source technologies. Through Oracle Cloud Infrastructure and its global cloud regions, we’re providing more opportunities than ever to make open source technology available to everyone.
```

Having a specific number of regions means we have to keep that number up-to-date.

## Get Started

> [Sign contributor agreement](https://oca.opensource.oracle.com/)

Reviewer: I'm not sure this needs to be here.

![Oracle Logo](ogho_banner.png "Open source at Oracle. Everyone is invited. Let's collaborate together")

# Open Source at Oracle

Oracle is the purveyor of the industry’s most widely adopted open source technologies, such as [Java](https://openjdk.java.net/) and [MySQL](https://www.mysql.com/), and one of the founding members of the [Linux Foundation](https://linuxfoundation.org/), the [Eclipse Foundation](https://www.eclipse.org/), and the [Java Community Process](https://jcp.org/en/home/index).

But what has guided us along the way?

Our mission is to empower developers and continue supporting open source technologies. Through Oracle Cloud Infrastructure and its 37 cloud regions, we’re providing more opportunities than ever to make open source technology available to everyone.<br><br>

## Get Started

[Learn about open source at Oracle](https://developer.oracle.com/open-source/)

[Join Oracle's developer community](https://developer.oracle.com/)<br><br>

## All Oracle GitHub organizations

> * [Oracle on GitHub](https://github.com/oracle)

Reviewer: This should just be Oracle

* [AddThis](https://github.com/addthis)
* [Fn Project](https://github.com/fnproject)
* [Java Community Process](https://github.com/jcp-org)
* [Java EE](https://github.com/javaee)
* [MySQL](https://github.com/mysql)
* [Oracle DevRel](https://github.com/oracle-devrel)
* [Oracle GitHub Actions](https://github.com/oracle-actions)

> * [Oracle LiveLabs](https://github.com/oracle-livelabs)

Reviewer: This org isn't ready yet, so can we not list it?

Author: ok

[Learn about open source at Oracle](https://developer.oracle.com/open-source/)

> [Join Oracle's developer community](https://bit.ly/odevrel_slack)<br><br>

Reviewer: ```suggestion
[Join Oracle's developer community](https://bit.ly/odevrel_slack) on Slack
```

Adding HTML in Markdown is not recommended. Also, isn't there an Oracle branded URL shortening service we can use instead of Bit.ly? One that can track how many clicks we get?

Author: noted about HTML tags. Re link, it's coming from corporate devrel and they are using it everywhere

## All Oracle GitHub organizations

> * [Oracle](https://github.com/oracle) – the main Oracle GitHub organization

Reviewer: ```suggestion
* [Oracle](https://github.com/oracle) – our primary GitHub organization
```

* [AddThis](https://github.com/addthis)- a social sharing platform
* [Fn Project](https://github.com/fnproject) – the container native, cloud agnostic serverless platform
* [Java Community Process](https://github.com/jcp-org) – developing standard technical specifications for Java technology
* [Java EE](https://github.com/javaee) – the Java Enterprise Edition platform
* [MySQL](https://github.com/mysql) –  the world's most popular open source database
* [Oracle DevRel](https://github.com/oracle-devrel) – developer code samples, tutorials, and more
* [Oracle GitHub Actions](https://github.com/oracle-actions) – GitHub Actions from Oracle

> * [Oracle Quick Start](https://github.com/oracle-quickstart) – automated deployments of enterprise software on OCI

Reviewer: ```suggestion
* [Oracle Quick Start](https://github.com/oracle-quickstart) – automated deployments of enterprise software on Oracle Cloud Infrastructure
```

* [Oracle Sample Projects](https://github.com/oracle-samples) – new projects and sample applications 

> * [Oracle Terraform Modules](https://github.com/oracle-terraform-modules) – Terraform modules for provisioning cloud resources

Reviewer: ```suggestion
* [Oracle Terraform Modules](https://github.com/oracle-terraform-modules) – Terraform modules for provisioning Oracle Cloud Infrastructure resources
```
